h/static/styles/partials-v2/_search.scss

```diff
@@ -172,7 +179,10 @@
 
   .search-result-bucket__annotation-cards-container {
     flex-direction: column;
-    padding-left: 0;
+  }
+
+  .search-result-bucket__annotation-cards {
+    width: 100%;
```

Comment:
Shrink annotation card on small (<500px) screens from 500px (wider than the screen) to 100% of the space available (ie as wide as the screen minus margins and padding)